pkg/virt-controller/watch/vm/vm_test.go

```diff
@@ -7010,6 +7010,55 @@ var _ = Describe("VirtualMachine", func() {
 			})
 		})
 
+		Context("feature configurables", func() {
+			It("should update the VM conditions if the downwardMetrics is disabled and it is requested in a stopped VM", func() {
+				vm, _ := watchtesting.DefaultVirtualMachine(false)
+				vm.Spec.Template.Spec.Domain.Devices.DownwardMetrics = &v1.DownwardMetrics{}
+
+				vm, err := virtFakeClient.KubevirtV1().VirtualMachines(vm.Namespace).Create(context.Background(), vm, metav1.CreateOptions{})
+				Expect(err).To(Succeed())
+				addVirtualMachine(vm)
+
+				sanityExecute(vm)
+
+				vm, err = virtFakeClient.KubevirtV1().VirtualMachines(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
+				Expect(err).To(Succeed())
```

Comment:
```suggestion
				Expect(err).ToNot(HaveOccured())
```

nit: personal preference

Reply:
Done.